test/js/pages.test.js

```diff
@@ -97,6 +98,15 @@ describe('page filters', () => {
     expect(testPage).toHaveProperty('year');
   });
 
+  test('eventSort', () => {
+    const slugs = ['test1', 'events', 'test3'];
+    const years = ['2020', '2019', '2018'];
+    const actual = eventSort(collection4);
+
+    expect(actual.map((item) => item.fileSlug)).toEqual(slugs);
+    expect(actual.map((item) => item.year)).toEqual(years);
+  });
```

Comment:
@jgerigmeyer @duggiemitchell I tried writing tests here, but they're failing, and I'm not sure why. Help? :)